mlshvch/hometask-5/1.sql

```diff
@@ -0,0 +1,3 @@
+SELECT CONCAT(first_name, ' ', last_name) AS actors
+FROM actor
+WHERE last_name = 'Zellweger';
```

Comment:
Let's build a healthy habit of leaving an empty newline in the end of each file.